service/history/execution/mutable_state_builder.go

```diff
@@ -2220,14 +2220,14 @@ func (e *mutableStateBuilder) ReplicateActivityTaskScheduledEvent(
 		TaskList:                 attributes.TaskList.GetName(),
 		HasRetryPolicy:           attributes.RetryPolicy != nil,
 	}
-	ai.ExpirationTime = ai.ScheduledTime.Add(time.Duration(scheduleToCloseTimeout) * time.Second)
+
```

Comment:
Where is this activity info used? I couldn't find in the code where this is used.

Reply:
Check `RetryActivity()` method in this file

Reply:
Oh, I see. Thanks.